class.jetpack.php

```diff
@@ -537,7 +537,7 @@ private function __construct() {
 		 * Prepare Gutenberg Editor functionality
 		 */
 		require_once JETPACK__PLUGIN_DIR . 'class.jetpack-gutenberg.php';
-		add_action( 'enqueue_block_assets', array( 'Jetpack_Gutenberg', 'enqueue_block_assets' ) );
```

Comment:
Seems like we'll no longer use core Gutenberg's hook `enqueue_block_assets` to enqueue the assets. Core Gutenberg recommends using that hook, because it assures that its necessary assets [will already be loaded](https://github.com/WordPress/gutenberg/blob/3b66a7f21f646a4a78379330f3ae4181ccefeac0/lib/client-assets.php#L1152). Do we want to use it after all, even if we use `init` for our business logic?

Reply:
I think we are fine with not using it. since the current enqueue_block_assets hood also fires on the front end and results in assets being loaded when they don't have to be.